### PR TITLE
chore: add node version hints

### PR DESCRIPTION
**Pull request by dickie81**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,16 +14,17 @@ You can try without these requirements, but you'd be on your own.
 
 ## Getting Started
 
-1. Run `yarn` to install dependencies
-2. components can be seen running in a storybook:
+1. Make sure you are using Node 8.x
```
Comment (craigbilner) [inline README.md]: might want to specify `npm` and `yarn` too as these both may cause issues?

Comment (dickie81) [inline README.md]: Not sure what versions of yarn / npm work & don't work. However promisify breaks node 6 as its not available which means Node < v8 definitely does not work. Jack & Aron both ran into this after master was merged into the brightcove-video branch.

Comment (craigbilner) [inline README.md]: Rohan had issues with `npm 5.0.0` so I would say at least `5.0.3` and yarn `27.5`...just cos it's latest

Comment (jackharvey1) [inline README.md]: not had any issues with `npm 3.10.10` fwiw

**Comment by dickie81** [comment]
please squash and merge...

**Comment by trevorah** [comment]
@dickie81 I've shuffled the text around a bit. is that ok?

**Comment by dickie81** [comment]
@trevorah no problem.

**Pull request merged**